alex/applications/PublicTransportInfoCS/hdc_policy.py

```diff
@@ -572,7 +572,32 @@ def get_requested_info(self, requested_slots, ds, accepted_slots):
         res_da = DialogueAct()
 
         for slot in requested_slots:
-            if ds['route_alternative'] in [0, 1, 2, 3]:
+            # for these, we don't need any route information, we can answer straight away
+            if slot not in ['from_stop', 'to_stop',
+                            'departure_time', 'departure_time_rel',
+                            'arrival_time', 'arrival_time_rel',
+                            'duration', 'num_transfers', 'time_transfers', ]:
+                dai = DialogueActItem("inform", slot, requested_slots[slot])
+                res_da.append(dai)
+                # remember that we answered
+                ds["rh_" + slot].reset()
+                continue
+
+            # try to find a route if we don't know it yet
+            if not isinstance(ds['route_alternative'], int):
```

Comment:
I don't know – the original code was written this way. But I think it makes sense in a way – you may ask, e.g., "how long does the journey take and when will I be there", and you should get both answers in the same turn.